### PR TITLE
add anchor.js@1.0.6 w/ git autoupdate

### LINKED ISSUE #6667: [Request] Add anchor.js
**Library name: anchor.js** 
**Git repository url: https://github.com/cbopp-art/anchor.git**
**npm package url(optional):** 
**License(s): GPL-2.0+**
**Official homepage: http://cbopp-art.github.io/anchor/**
**Wanna say something? Leave message here: Hi, thanks a lot for a fast include, have open issues asking for this ;)**

=====================
Notes from cdnjs maintainer:
You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by Piicksarn**
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): **https://github.com/cbopp-art/anchor**
 * Official website (optional, not the repository): **http://cbopp-art.github.io/anchor/**
 * NPM package url (optional):
 * GitHub / Bitbucket popularity (required): 
   - Count of stars: **34**
   - Count of watchers: **4**
   - Count of forks: **9**
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [x] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #6667, cc @cbopp-art

### CODE REVIEW

**Comment by Piicksarn** [comment]
@Amomo, would you please help me for checkout the pr?
Thank you!

**Comment by Piicksarn** [comment]
@maruilian11, would you mind to help me for checking the pr? 
thank you!!

File: ajax/libs/anchor.js/package.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  "keywords": [
+    "jquery-plugin",
+    "section",
+    "link",
+    "anchor",
+    "animation"
+  ],
+  "version": "1.0.6",
+  "author": "Cornel Boppart (https://github.com/cbopp-art)",
+  "license": "deprecated_GPL-2.0+",
```
Comment (maruilian11) [inline ajax/libs/anchor.js/package.json]: can you tell me how you told it's `deprecated_GPL-2.0+` not `GPL-3.0`? i saw [GPL-3.0](https://github.com/cbopp-art/anchor/blob/master/license.txt#L4) here.

Comment (Piicksarn) [inline ajax/libs/anchor.js/package.json]: https://github.com/cbopp-art/anchor/blob/master/package.json#L16

Comment (maruilian11) [inline ajax/libs/anchor.js/package.json]: i knew that, but if there is any uncertain object, we should at least open an issue then send the PR.

**Comment by PeterDaveHello** [comment]
Ping.

**Comment by PeterDaveHello** [comment]
Ping.

**Comment by Piicksarn** [comment]
@PeterDaveHello , i followed [license.txt](https://github.com/cbopp-art/anchor/blob/master/license.txt#L4) and modified the license part in package.json
Would you check it again? 
Thanks

**Comment by PeterDaveHello** [comment]
@Piicksarn I'm okay with `"deprecated_GPL-2.0+"`, have no idea about the reason to use `"GPL-3.0"` here, have you confirmed with the author?

**Comment by Piicksarn** [comment]
@PeterDaveHello, sure!

**Comment by PeterDaveHello** [comment]
@Piicksarn Let's rollback first, I don't want to have so many PR opened for a long time.

**Comment by PeterDaveHello** [comment]
@Piicksarn use "deprecated_GPL-2.0+" first.

**Comment by Piicksarn** [comment]
@PeterDaveHello , i have it modified and need u for taking a look! Thanks!

**Comment by PeterDaveHello** [comment]
Okay, finally ...

**Pull request merged**